### DISCUSSION TITLE
Share images

### DISCUSSION THREAD

**Discussion by ElenaTimiakTech** [MurhafSousli/ngx-sharebuttons#592] [Q&A] [answered]
Is it possible to share images with any social network? I have read that it can be done on pinterest but I am interested in all the others as well.

**Comment by MurhafSousli** [comment, marked as answer]
Exactly, all the others don't allow passing the images as URL parameters, the only way that works is by having a meta tag for the image

e.g.
```html
<meta property="og:image" content="http://euro-travel-example.com/thumbnail.jpg">
```

So you should get your app SEO friendly using Angular Universal
